sc-deployer/tests/index.ts

```diff
@@ -0,0 +1,16 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { Args, deploySC, IAccount, ISCData } from '../dist/index';
+
+const account: IAccount = {
+    secretKey: "S126gPee3LJwrQEPqukJ26f6bHG8t5GPwH1WHhR9VpyqrkhkLCGf",
```

Comment:
warning private key here

does this test file meant to deploy something into the blockchain? it should target a sandbox node

Reply:
Done.